### ISSUE TITLE
Add routine for typing the "pair" types

### ISSUE THREAD

**Issue by jennyfothergill** [rsdefever/GAFF-foyer#12]
I'm opening this issue mainly to start a discussion on how typing group III types such as (cc/cd) should be handled. 
It seems to be the there could be some sort of logic for walking along a conjugated section and changing the types, but that could be difficult to implement as a plugin.

**Comment by rsdefever** [comment]
I agree, it seems doable in principle. I spent some time trying to come up with an algorithm a bit over a year ago, but I never came up with anything good and eventually "gave up". I'm open to suggestions if this is something others are interested in pursuing.
